python/bootstrap.py

```diff
@@ -336,7 +336,7 @@ def setup_projects(backend):
 
```

Comment:
If you take a look at lines 177 - 277 of the bootstrap.py file you can see that there are a bunch of listed permissions. These are the fusion pipelines external applications (namely searchhub) has permission to access. If we ever want to be able to roll this pipeline out into production you will need to add GET access against the pipeline you have added.